jobs/sportpursuit-python-developers-x2-roles.html

```diff
@@ -0,0 +1,70 @@
+    email: join@sportpursuit.co.uk
+    phone: # Phone number (optional)
+    appliction form: https://sportpursuit.peoplehr.net/Pages/JobBoard/Opening.aspx?v=3b21579e-16a4-4828-941e-91ac3f4ddc5b
+created: !!timestamp 2018-04-25
+tags:
+  - python
+  - sql
+  - django
+  # include relevant tags
+---
+
+# Python Developer (x2 roles)
+
+## Overview
+SportPursuit is a leading private shopping community for over 4 million active men and women across UK and Europe. We were rated as the 5th fastest growing business in the UK (The Times Sage Startup Track 15 2015) and offer a culture where self-starters have an opportunity to have a huge impact.
+
+Our mission is to offer amazing offers on the best sports products and services from the world’s leading sports brands to our members in week long ‘flash’ sales. Our deals are only available to our members behind our sign in wall; this gives our brands the confidence to deliver exclusive offers to our members whilst protecting their brands and their full price retailers from continual discounting.
+
+Our offices are located at Workspace’s flagship hub in Kennington Park next to the Oval Cricket ground and recently SportPursuit has become a member of the Future Fifty, the UK's leading programme for late-stage digital businesses.
+
+If you’re ambitious, talented, and want to shape the future of the business but don’t mind getting your hands dirty this could be just the place for you.
+
+## Summary
+We are looking for a 2 x Python Developers to assist with development on some key areas of our internal technology platform, namely our Python based ERP platform and Business Intelligence / Data Warehouse system.
+
+## Responsibilities
+* Work in a scrum / agile environment to help develop key features to improve business functionality
+* Backend Python development and provide technical advice to development team members
+* Assist with the design and architecture of the existing data warehouse, with the aim to centralise customer, product and order data.
+* Assist with the technical design and architecture of integration between existing e-commerce platform, ERP and external suppliers.
+* Expand the existing website API capability, specifically around product recommendation and personalisation
+* Expand the existing reporting capabilities to support day-to-day operation of the business.
+* Work alongside internal developers and vendors to customise ERP system to suite business needs
+* Work with key suppliers to build integration to stock and product feeds.
+* Enhance the level of site personalisation, by improving both sport and product recommendation logic.
+* Enhance the machine learning capability for automatic categorisation and tagging of products.
+
+## Skills and Experience
+* Graduate in computer science or related discipline
+* 2-5 years of object orientated Python development experience
+* Development experience using open source Python frameworks ideally Django
+* Exposure to Odoo (OpenERP) ERP platform
```

Comment:
This seems like a very specific requirement - you should probably highlight this in the tags section. Consider changing the job title to be something like "OpenERP / Odoo developer", since this is an unusual skill to have.